spring-cloud-gcp-data-spanner/src/test/java/org/springframework/cloud/gcp/data/spanner/core/mapping/SpannerConvertersTest.java

```diff
@@ -34,9 +36,16 @@
 
 	@Test
 	public void dateConversionTest() {
-		Date date = Date.fromYearMonthDay(2018, 3, 29);
-		assertThat(SpannerConverters.JAVA_TO_SPANNER_DATE_CONVERTER
-				.convert(SpannerConverters.SPANNER_TO_JAVA_DATE_CONVERTER.convert(date))).isEqualTo(date);
+		Timestamp timestamp = Timestamp.now();
+		assertThat(SpannerConverters.DATE_TIMESTAMP_CONVERTER
+				.convert(SpannerConverters.TIMESTAMP_DATE_CONVERTER.convert(timestamp))).isEqualTo(timestamp);
+	}
+
+	@Test
+	public void localDateConversionTest() {
+		LocalDate localDate = LocalDate.now();
```

Comment:
Can we also have similar tests for pre-epoch timestamps, since we got burned by those already?